src/components/contact/contactMessage/ContactMessage.css.ts

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components';
+import colors from '../../../styles/colors';
+// import Button from "../../common/button/Button";
+import { ButtonWrapperProps } from '../../common/button/Button.css';
+
+/* #FFFFFF for white panels and CONTACT text */
```

Comment:
Let's copy this to a DEVELOPMENT.md; that way it'll be in one place that future developers can use.